docs/user/azure/install_upi.md

```diff
@@ -344,7 +344,7 @@ Create the deployment using the `az` client:
 
 ```sh
 export BOOTSTRAP_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c "files" -n "bootstrap.ign" -o tsv`
-export BOOTSTRAP_IGNITION=`jq -rcnM --arg v "3.1.0" --arg url $BOOTSTRAP_URL '{ignition:{version:$v,config:{replace:{source:$url}}}}' | base64 -w0`
+export BOOTSTRAP_IGNITION=`jq -rcnM --arg v "3.1.0" --arg url $BOOTSTRAP_URL '{ignition:{version:$v,config:{replace:{source:$url}}}}' | base64 | tr -d '\n'`
```

Comment:
if we look at the docs for base64 https://linux.die.net/man/1/base64

```
-w, --wrap=COLS
    Wrap encoded lines after COLS character (default 76). Use 0 to disable line wrapping. 
```

`-w0` already does what we need, we should just use that right? instead of another utility to remove newlines.

Reply:
This is a Linux only feature, it doesn't work on Mac OS or BSD. I know we have at least a few Mac OS users. What is wrong with keeping this platform neutral ?

Reply:
That makes sense. I think in that case your solution works better.